bundle/android/strings.xml

```diff
@@ -142,7 +142,7 @@
     <string name="gutenberg_native_duplicate_block" tools:ignore="UnusedResources">Duplicate block</string>
     <string name="gutenberg_native_each_block_has_its_own_settings_to_find_them_tap_on_a_block_its_s" tools:ignore="UnusedResources">Each block has its own settings. To find them, tap on a block. Its settings will appear on the toolbar at the bottom of the screen.</string>
     <string name="gutenberg_native_edit_cover_media" tools:ignore="UnusedResources">Edit cover media</string>
-    <string name="gutenberg_native_edit_file" tools:ignore="UnusedResources">Edit file</string>
+    <string name="gutenberg_native_edit_file" tools:ignore="UnusedResources">Edit file </string>
```

Comment:
Not sure if this is accurate, this string is referenced [here](https://github.com/WordPress/gutenberg/blob/1f1bccf711a9f783881ea51b3f0bf6b32c4ebc18/packages/block-library/src/file/edit.native.js#L232) but it doesn't include the extra space 🤔 .